### DISCUSSION TITLE
code length overflow. (2196 > 2192 bit)

### DISCUSSION THREAD

**Discussion by wadealer** [chillerlan/php-qrcode#238] [Feedback]
When QR code rendered exception acquired `code length overflow. (2196 > 2192 bit)`. 

`$config = [
		'level'		=> 40,
		'eccLevel'	=> EccLevel::L,
	];`

What should I do to avoid this error?

**Comment by codemasher** [comment]
Hi, first off, the option "level" doesn't exist - i assume you mean "version". Second, it seems to me that the contrary to the above snippet, the version is fixed/limited to 10 (which has a maximum capacity of 2192 bits, see [Information capacity and versions of the QR Code](https://www.qrcode.com/en/about/version.html)). Simply remove the "version" setting, so that auto detection can work.
